README.md

```diff
@@ -1,3 +1,5 @@
+> This fork just offers a simple enhancement to post metadata. Here's [the PR](https://github.com/mmistakes/minimal-mistakes/pull/5014).
```

Comment:
Is this line intended to come up in the PR?

Reply:
Ah no sorry! I'll kill it.

Reply:
reverted